### PR TITLE
Minor fixed found during demo preparation

### PR DESCRIPTION
**Pull request by npinaeva**
Use get_full_path for file inputs to make sure we convert relative path correctly.
Only use bash -c with braced commands for netnamespace
Add contacts in README

Signed-off-by: Nadia Pinaeva <npinaeva@redhat.com>

### CODE REVIEW

File: debug-scripts/utils [hunk 1]
```diff
@@ -100,11 +103,17 @@ nsenter -n -t $ns_pid <command>
...
         FULL_COMMAND="($FULL_COMMAND) $SLEEP"
       fi
       oc debug node/"$node_name" --image=$NETWORK_TOOLS_IMAGE -- bash -c "$FULL_COMMAND"
     fi
 }
 
+get_full_path() {
+  local_path=$1
+  pushd $NETWORK_TOOLS_INITIAL_DIR > /dev/null || exit 1
+  echo "$(cd "$(dirname "$local_path")"; pwd)/$(basename "$local_path")"
```
[the remark this answers is not in the document]
Comment (flavio-fernandes) [reply debug-scripts/utils:117]: I second what @andreaskaris says here, @npinaeva . pushd seems a bit of overkill.
How about something like:
```
get_full_path() {
    FP="${NETWORK_TOOLS_INITIAL_DIR}/$1"
    readlink -e "$FP" || { >&2 echo "Bad path $FP"; exit 1; }
}
```

**Review by flavio-fernandes — CHANGES_REQUESTED** [review]
some ideas for improving full path. otherwise, lgtm.

**Comment by npinaeva** [comment]
You are absolutely right, I was too lazy to do that properly
Thanks for paying attention! PTAL if this update is better (notice that this function should also work with absolute path)

File: debug-scripts/utils [hunk 2]
```diff
@@ -100,11 +103,19 @@ nsenter -n -t $ns_pid <command>
...
+      FULL_COMMAND="nsenter -n -t $ns_pid $with_bash $command"
       if [ -n "$SLEEP" ]; then
         FULL_COMMAND="($FULL_COMMAND) $SLEEP"
       fi
       oc debug node/"$node_name" --image=$NETWORK_TOOLS_IMAGE -- bash -c "$FULL_COMMAND"
     fi
 }
 
+get_full_path() {
+  if [[ "$1" = /* ]]; then
```
Comment (flavio-fernandes) [inline debug-scripts/utils:115]: nice!

**Review by flavio-fernandes — COMMENTED** [review]
/lgtm

**Pull request merged**